ukrainian/eat-wall/eat-wall.md

```diff
@@ -0,0 +1 @@
+Зефір виявився не тільки їстівним, але й дуже смачним. Ви відчуваєте, що доволі сильно зголодніли і це відмінна можливість виправити становіще.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.